src/analysis/calculus/affine_map.lean

```diff
@@ -33,4 +33,34 @@ begin
   exact times_cont_diff_const,
 end
 
+/-- Note that in the case `X = (V →A[𝕜] W) × V`, `f = prod.fst`, `g = prod.snd` this is the
+statement that evaluation map on the space of continuous affine maps is smooth.
+See Note [continuity lemma statement] for why we parameterise by `X`. -/
+lemma times_cont_diff_apply {X : Type*} [normed_group X] [normed_space 𝕜 X] {n : with_top ℕ}
```

Comment:
Should this be in the `times_cont_diff` namespace instead, and use dot notation?